src/components/activity/ActivityFormPanel.svelte

```diff
@@ -167,52 +249,158 @@
+          </fieldset>
+
+          <fieldset>
+            <label for="simulationStatus">Simulation Status</label>
+            <input
+              class="st-input w-100"
+              disabled
+              name="simulationStatus"
+              value={unfinished ? 'Unfinished' : duration ? 'Finished' : 'None'}
+            />
+          </fieldset>
+
+          <DatePickerField
+            disabled={isChild}
+            field={startTimeField}
+            label="Start Time - YYYY-DDDThh:mm:ss"
+            name="start-time"
+            on:change={onUpdateStartTime}
+            on:keydown={onUpdateStartTime}
+          />
+
+          {#if duration !== null}
+            <fieldset>
+              <label for="endTime">End Time</label>
+              <input class="st-input w-100" disabled name="endTime" value={endTime} />
+            </fieldset>
+          {/if}
+
+          <fieldset>
+            <label for="creationTime">Creation Time</label>
+            <input class="st-input w-100" disabled name="creationTime" value={creationTime ?? 'None'} />
+          </fieldset>
+
+          <fieldset>
+            <label for="lastModifiedTime">Last Modified Time</label>
+            <input class="st-input w-100" disabled name="lastModifiedTime" value={lastModifiedTime ?? 'None'} />
+          </fieldset>
+
+          <fieldset>
+            <label for="sourceSchedulingGoalId">Source Scheduling Goal ID</label>
+            <input
+              class="st-input w-100"
+              disabled
+              name="sourceSchedulingGoalId"
+              value={sourceSchedulingGoalId ?? 'None'}
+            />
+          </fieldset>
+
+          {#if duration !== null}
+            <fieldset>
+              <label for="endTime">End Time</label>
+              <input class="st-input w-100" disabled name="endTime" value={endTime} />
+            </fieldset>
+          {/if}
+
+          <fieldset>
+            <label for="activityTags">Tags</label>
+            <!--
+              Make use of svelte's 'key' here and switch on activity ID in order to clear the
+              text input in the tags component which would otherwise remain populated with
+              dirty input when a user switches to a new activity.
+             -->
+            {#key id}
+              <Tags
+                disabled={isChild}
+                autocompleteValues={planTags}
+                name="activityTags"
+                on:change={onUpdateTags}
+                {tags}
+              />
+            {/key}
+          </fieldset>
+        </details>
+      </div>
 
       <fieldset>
-        <label for="simulationStatus">Simulation Status</label>
-        <input
-          class="st-input w-100"
-          disabled
-          name="simulationStatus"
-          value={unfinished ? 'Unfinished' : duration ? 'Finished' : 'None'}
-        />
+        <details open style:cursor="pointer">
+          <summary>Annotations</summary>
```

Comment:
Can we have a case in here where if there are no annotations (array is empty) we display 'No Annotations Found'?

Reply:
Agreed, adding.